### ISSUE TITLE
Problème d'installation

### ISSUE THREAD

**Issue by tsoonav** [afimb/siri_lite#1]
Bonjour,
J'ai commencé à installer et j'ai un soucis au moment du déploiement avec la commande : 
`mvn -DskipTests install` 
Le message d'erreur indique que cela vient de uk.org.siri.wsdl

Y a t il des choses à configurer spécifiquement ?
Merci par avance

**Comment by metienne** [comment]
Bonjour, 
pouvez-vous préciser votre environnement ainsi que le message d'erreur reçu? 
Cordialement

**Comment by tsoonav** [comment]
Merci pour la réponse rapide.
Je suis sur un environnement de test sur un ubuntu server 16.04.2 LTS
j'utilise openjdk

Je n'ai l'erreur suivante :

> Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.2:compile (default-compile) on projetc uk.org.siri.wsdl : Error while storing the mojo status : /home/jerome/siri_lite/uk.org.siri.wsdl/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst (Permission denied)

**Comment by metienne** [comment]
Il semble que vous avez un problème de droit : Permission denied

**Comment by tsoonav** [comment]
Il semble que tous les droits sont ouverts sur le dossier qui pose problème

drwxrwxr-x, c'est bien l'équivalent d'un chmod 777 ?

Après je ne suis pas expert sur l'administration d'un serveur linux

**Comment by metienne** [comment]
non, c'est un 775 ; seul l'utiisateur et le groupe ont accès en écriture; 
utilisez-vous un autre compte que celui (jerome) propriétaire du répertoire pour compiler? 
si oui, est-ce bien nécessaire? 
les répertoires de résultat de compilation (target/*) sont créés par maven avec des droits par défaut qui ne sont certainement pas compatible d'une telle manipulation.

**Comment by tsoonav** [comment]
Non je n'utilise pas d'autre compte, je n'ai que celui-ci configuré sur la machine (+celui créé pour windfly)
J'essaye de lancer en sudo pour voir.
J'ai maintenant une erreur sur siri_lite.server

Est ce que je dois renseigner les adresses des serveurs SIRI à lire ici ?

**Comment by metienne** [comment]
pouvez-vous me contacter directement par mail metienne(at)cityway.fr ?
